integration-tests/gradle/src/test/java/io/quarkus/gradle/FastJarFormatWorksTest.java

```diff
@@ -20,6 +33,55 @@ public void testFastJarFormatWorks() throws Exception {
         assertThat(quarkusApp).exists();
         Path jar = quarkusApp.resolve("quarkus-run.jar");
         assertThat(jar).exists();
+
+        File output = new File(projectDir, "build/output.log");
+        output.createNewFile();
+        Process process = launch(jar, output);
+        try {
+            // Wait until server up
+            dumpFileContentOnFailure(() -> {
+                await()
+                        .pollDelay(1, TimeUnit.SECONDS)
+                        .atMost(1, TimeUnit.MINUTES).until(() -> DevModeTestUtils.getHttpResponse("/hello", 200));
+                return null;
+            }, output, ConditionTimeoutException.class);
+
+            String logs = FileUtils.readFileToString(output, "UTF-8");
+
+            assertThatOutputWorksCorrectly(logs);
+
+            // test that the application name and version are properly set
+            assertThat(DevModeTestUtils.getHttpResponse("/hello",getQuarkusDevBrokenReason()).equals("hello"));
+        } finally {
+            process.destroy();
+        }
+    }
+
+    static void assertThatOutputWorksCorrectly(String logs) {
+        assertThat(logs.isEmpty()).isFalse();
+        String infoLogLevel = "INFO";
+        assertThat(logs.contains(infoLogLevel)).isTrue();
+        assertThat(logs.contains("cdi, resteasy")).isTrue();
+    }
+
+    private void dumpFileContentOnFailure(final Callable<Void> operation, final File logFile,
+                                          final Class<? extends Throwable> failureType) throws Exception {
+
+        final Logger log = Logger.getLogger(FastJarFormatWorksTest.class);
+        try {
+            operation.call();
+        } catch (Throwable t) {
+            log.error("Dumping logs that were generated in " + logFile + " for an operation that resulted in "
+                        + t.getClass().getName() + ":",t);
+
+            throw t;
+        }
+    }
+
+    private static Supplier<String> getQuarkusDevBrokenReason() {
+        return () -> {
+            return quarkusDev == null ? null : quarkusDev.isDone() ? "quarkusDev mode has terminated" : null;
```

Comment:
And the string should be `"jar run mode has terminated"`

Reply:
Okay